### ISSUE TITLE
Scale to fit automatically?

### ISSUE THREAD

**Issue by arashster** [mileszs/wicked_pdf#1002]
Is it possible to have wicked_pdf "scale to fit" the generated PDF to a single page (specifying the page size, e.g. Letter, A4, etc)?

**Comment by vpanmand-mdsol** [comment]
Can we only print one page and ignore the rest of the pages using wicked_pdf??

**Comment by unixmonkey** [comment]
Not really. You can play with the `zoom` option, or setup your HTML document to be exactly the right size for both width and height, but it'll be quite hard to get right and also be reproducible.

As an alternative, you might be able to use `wkhtmltoimage` to make an image of the HTML, then scale that to the size of a single page, and embed that into a PDF.

If you only care about the first page, you could create the PDF, then use a PDF manipulation tool to make a new PDF from only the first page.

I don't know your use-case, but I'd suggest either letting it do it's thing, or use a real PDF generation tool, like Prawn if possible, or a service like DocRaptor.

Please let me know what might work for you and how it goes. I'm here to help if I can.
